TestApp/app/screens/AppCenterScreen.js

```diff
@@ -225,9 +225,17 @@ export default class AppCenterScreen extends Component {
               data: [
                 { title: 'Install ID', value: 'installId' },
                 { title: 'SDK Version', value: 'sdkVersion' },
+                {
+                  title: 'User ID',
+                  value: 'userId',
+                  onChange: async (userId) => {
```

Comment:
Why there are two `onChange`?

Reply:
won't fix.

Reply:
It's not duplicated. https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/504/files#diff-9be630c1ef08339c52b5f05a2fa83720R149 is inlined function definition and https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/504/files#diff-9be630c1ef08339c52b5f05a2fa83720R231 is the parameters passed into the function.